# mayhem_program::entry  (yours 1888 B at .text+0x6f300; original 1968 B at .text+0x7d710)
# distance 213 instructions; evidence: same-callees 21/21; neighbours between the same matched functions as in your build
--- original
+++ yours
@@ -10,125 +10,117 @@
 ldxdw r0, [r1+24]
 lddw r6, 0xa3653d805945034e
 jne r0, r6, +0x140
-jlt r5, 8, +0x528
+jlt r5, 8, +0x4d8
 ldxdw r0, [r4]
-lddw r6, 0x1d9acb512ea545e3
-jsgt r0, r6, +0x150
-lddw r6, 0xcb89540fb73025be
-jsle r0, r6, +0x1e8
-lddw r6, 0xe4b3b83cd528b0a0
-jsle r0, r6, +0x2e8
-lddw r6, 0xe7759c6fe1c5934b
-jsgt r0, r6, +0x488
-lddw r6, 0xe4b3b83cd528b0a1
-jeq r0, r6, +0x5b0
-lddw r6, 0xe53e4896cbc266ea
-jne r0, r6, +0x528
-add64 r4, 8
-add64 r5, -8
-call mayhem_program::__private::__global::extend_account#20dc
-ja +0x538
-mov64 r1, 4100
-ja +0x530
+lddw r6, 0xeaebda01123d0665
+jsle r0, r6, +0x150
 lddw r6, 0x5191086c079d93dd
 jsle r0, r6, +0x268
 lddw r6, 0x676fc47d8b4c4b7f
-jsle r0, r6, +0x350
+jsle r0, r6, +0x420
 lddw r6, 0x6f727af23b6ab07e
-jsgt r0, r6, +0x4d8
+jsgt r0, r6, +0x488
 lddw r6, 0x676fc47d8b4c4b80
-jeq r0, r6, +0x5d0
+jeq r0, r6, +0x6e0
 lddw r6, 0x6bda9e89bc871b1e
-jne r0, r6, +0x528
+jne r0, r6, +0x4d8
 add64 r4, 8
 add64 r5, -8
 call mayhem_program::__private::__global::withdraw_admin#e357
-ja +0x538
+ja +0x4e8
+mov64 r1, 4100
+ja +0x4e0
+lddw r6, 0xcb89540fb73025be
+jsgt r0, r6, +0x1e8
 lddw r6, 0xad837f01a485e632
-jsgt r0, r6, +0x3b8
+jsgt r0, r6, +0x2e8
 lddw r6, 0x8b94db282a747691
-jeq r0, r6, +0x670
+jeq r0, r6, +0x5e0
 lddw r6, 0x92e185c805a7ad22
-jeq r0, r6, +0x730
+jeq r0, r6, +0x560
 lddw r6, 0x942242764762e1af
-jne r0, r6, +0x528
+jne r0, r6, +0x4d8
 add64 r4, 8
 add64 r5, -8
 call mayhem_program::__private::__global::sweep_fees#1513
-ja +0x538
+ja +0x4e8
+lddw r6, 0xe4b3b83cd528b0a0
+jsgt r0, r6, +0x350
+lddw r6, 0xcb89540fb73025bf
+jeq r0, r6, +0x600
+lddw r6, 0xcb8965eef091c610
+jeq r0, r6, +0x580
+lddw r6, 0xdacc283bea7f120d
+jne r0, r6, +0x4d8
+add64 r4, 8
+add64 r5, -8
+call mayhem_program::__private::__global::initialize_global_params#7917
+ja +0x4e8
 lddw r6, 0x3dd3c56761ee17b7
-jsgt r0, r6, +0x420
+jsgt r0, r6, +0x3b8
+lddw r6, 0xeaebda01123d0666
+jeq r0, r6, +0x6a0
 lddw r6, 0x1d9acb512ea545e4
-jeq r0, r6, +0x690
-lddw r6, 0x2dcc4555485ebd9d
-jeq r0, r6, +0x750
+jeq r0, r6, +0x660
 lddw r6, 0x39c4bcf0a29dadb0
-jne r0, r6, +0x528
+jne r0, r6, +0x4d8
 add64 r4, 8
 add64 r5, -8
 call mayhem_program::__private::__global::reset_sol_deployed#e67e
-ja +0x538
-lddw r6, 0xcb89540fb73025bf
-jeq r0, r6, +0x630
-lddw r6, 0xcb8965eef091c610
-jeq r0, r6, +0x6f0
-lddw r6, 0xdacc283bea7f120d
-jne r0, r6, +0x528
+ja +0x4e8
+lddw r6, 0xad837f01a485e633
+jeq r0, r6, +0x620
+lddw r6, 0xb240e9e73c82f65d
+jeq r0, r6, +0x5a0
+lddw r6, 0xc816c5cec88e57da
+jne r0, r6, +0x4d8
 add64 r4, 8
 add64 r5, -8
-call mayhem_program::__private::__global::initialize_global_params#7917
-ja +0x538
+call mayhem_program::__private::__global::initialize_fee_recipient#6652
+ja +0x4e8
+lddw r6, 0xe4b3b83cd528b0a1
+jeq r0, r6, +0x640
+lddw r6, 0xe53e4896cbc266ea
+jeq r0, r6, +0x5c0
+lddw r6, 0xe7759c6fe1c5934c
+jne r0, r6, +0x4d8
+add64 r4, 8
+add64 r5, -8
+call mayhem_program::__private::__global::sweep_fees_admin#8a63
+ja +0x4e8
+lddw r6, 0x3dd3c56761ee17b8
+jeq r0, r6, +0x6c0
+lddw r6, 0x411e9f4556d42e92
+jeq r0, r6, +0x680
+lddw r6, 0x4279d7e5fe330f4c
+jne r0, r6, +0x4d8
+add64 r4, 8
+add64 r5, -8
+call mayhem_program::__private::__global::burn_tokens#49c3
+ja +0x4e8
 lddw r6, 0x5191086c079d93de
-jeq r0, r6, +0x650
+jeq r0, r6, +0x740
 lddw r6, 0x59f83a72302a71cc
-jeq r0, r6, +0x710
+jeq r0, r6, +0x720
 lddw r6, 0x60af56654063cfc9
-jne r0, r6, +0x528
+jne r0, r6, +0x4d8
 add64 r4, 8
 add64 r5, -8
 call mayhem_program::__private::__global::update_trader#91a9
-ja +0x538
-lddw r6, 0xad837f01a485e633
-jeq r0, r6, +0x6b0
-lddw r6, 0xb240e9e73c82f65d
-jeq r0, r6, +0x770
-lddw r6, 0xc816c5cec88e57da
-jne r0, r6, +0x528
-add64 r4, 8
-add64 r5, -8
-call mayhem_program::__private::__global::initialize_fee_recipient#6652
-ja +0x538
-lddw r6, 0x3dd3c56761ee17b8
-jeq r0, r6, +0x6d0
-lddw r6, 0x411e9f4556d42e92
-jeq r0, r6, +0x790
-lddw r6, 0x4279d7e5fe330f4c
-jne r0, r6, +0x528
-add64 r4, 8
-add64 r5, -8
-call mayhem_program::__private::__global::burn_tokens#49c3
-ja +0x538
-lddw r6, 0xe7759c6fe1c5934c
-jeq r0, r6, +0x5f0
-lddw r6, 0xeaebda01123d0666
-jne r0, r6, +0x528
-add64 r4, 8
-add64 r5, -8
-call mayhem_program::__private::__global::buy#b519
-ja +0x538
+ja +0x4e8
 lddw r6, 0x6f727af23b6ab07f
-jeq r0, r6, +0x610
+jeq r0, r6, +0x700
 lddw r6, 0x7ef9e2596cde2492
-jne r0, r6, +0x528
+jne r0, r6, +0x4d8
 add64 r4, 8
 add64 r5, -8
 call mayhem_program::__private::__global::update_global_params#2db4
-ja +0x538
+ja +0x4e8
 mov64 r1, 101
 call <anchor_lang::error::Error as core::convert::From<anchor_lang::error::ErrorCode>>::from#ae9e
-jne r0, 2, +0x550
+jne r0, 2, +0x500
 mov64 r0, 26
-ja +0x5a8
+ja +0x558
 stxdw [r10-16], r0
 stxdw [r10-8], r1
 mov64 r2, r10
@@ -143,65 +135,65 @@
 exit
 add64 r4, 8
 add64 r5, -8
+call mayhem_program::__private::__global::update_max_sol_per_sweep#a93f
+ja +0x4e8
+add64 r4, 8
+add64 r5, -8
+call mayhem_program::__private::__global::sell_pump_swap#0df5
+ja +0x4e8
+add64 r4, 8
+add64 r5, -8
+call mayhem_program::__private::__global::sell_v2#bd0e
+ja +0x4e8
+add64 r4, 8
+add64 r5, -8
+call mayhem_program::__private::__global::extend_account#20dc
+ja +0x4e8
+add64 r4, 8
+add64 r5, -8
+call mayhem_program::__private::__global::reset_state#c06c
+ja +0x4e8
+add64 r4, 8
+add64 r5, -8
+call mayhem_program::__private::__global::reset_stable_deployed#919a
+ja +0x4e8
+add64 r4, 8
+add64 r5, -8
+call mayhem_program::__private::__global::sell#c304
+ja +0x4e8
+add64 r4, 8
+add64 r5, -8
 call mayhem_program::__private::__global::update_admin#d733
-ja +0x538
+ja +0x4e8
+mov64 r1, r2
+mov64 r2, r3
+call mayhem_program::__private::__events::__event_dispatch#2663
+ja +0x4e8
+add64 r4, 8
+add64 r5, -8
+call mayhem_program::__private::__global::resume_trading#f909
+ja +0x4e8
+add64 r4, 8
+add64 r5, -8
+call mayhem_program::__private::__global::buy#b519
+ja +0x4e8
+add64 r4, 8
+add64 r5, -8
+call mayhem_program::__private::__global::buy_v2#a800
+ja +0x4e8
 add64 r4, 8
 add64 r5, -8
 call mayhem_program::__private::__global::update_max_stable_per_sweep#6f25
-ja +0x538
+ja +0x4e8
 add64 r4, 8
 add64 r5, -8
-call mayhem_program::__private::__global::sweep_fees_admin#8a63
-ja +0x538
+call mayhem_program::__private::__global::stop_trading#735b
+ja +0x4e8
 add64 r4, 8
 add64 r5, -8
-call mayhem_program::__private::__global::resume_trading#f909
-ja +0x538
-add64 r4, 8
-add64 r5, -8
-call mayhem_program::__private::__global::reset_stable_deployed#919a
-ja +0x538
+call mayhem_program::__private::__global::withdraw_quote_admin#a6cc
+ja +0x4e8
 add64 r4, 8
 add64 r5, -8
 call mayhem_program::__private::__global::buy_pump_swap#1e53
-ja +0x538
-add64 r4, 8
-add64 r5, -8
-call mayhem_program::__private::__global::reset_state#c06c
-ja +0x538
-mov64 r1, r2
-mov64 r2, r3
-call mayhem_program::__private::__events::__event_dispatch#2663
-ja +0x538
-add64 r4, 8
-add64 r5, -8
-call mayhem_program::__private::__global::sell#c304
-ja +0x538
-add64 r4, 8
-add64 r5, -8
-call mayhem_program::__private::__global::buy_v2#a800
-ja +0x538
-add64 r4, 8
-add64 r5, -8
-call mayhem_program::__private::__global::sell_pump_swap#0df5
-ja +0x538
-add64 r4, 8
-add64 r5, -8
-call mayhem_program::__private::__global::withdraw_quote_admin#a6cc
-ja +0x538
-add64 r4, 8
-add64 r5, -8
-call mayhem_program::__private::__global::update_max_sol_per_sweep#a93f
-ja +0x538
-add64 r4, 8
-add64 r5, -8
-call anchor_lang::error::Error::with_values#ed6c
-ja +0x538
-add64 r4, 8
-add64 r5, -8
-call mayhem_program::__private::__global::sell_v2#bd0e
-ja +0x538
-add64 r4, 8
-add64 r5, -8
-call mayhem_program::__private::__global::stop_trading#735b
-ja +0x538
+ja +0x4e8
